Dockerfile

```diff
@@ -4,42 +4,67 @@ FROM rust:1.79.0-bookworm as builder
 
 ARG TARGETPLATFORM
 ARG GIT_V_VERSION
-ARG ONNXRUNTIME_VERSION=1.17.0
+ARG ONNXRUNTIME_VERSION=1.19.2
 ARG PROFILE=release
 ARG FEATURES
 
-RUN apt-get update && apt-get install -y llvm-dev libclang-dev clang cmake binutils
-
+RUN apt-get update && apt-get install -y llvm-dev libclang-dev clang cmake
 WORKDIR /usr/src/edge-runtime
 
+RUN --mount=type=cache,target=/usr/local/cargo/registry,id=${TARGETPLATFORM} \
+  cargo install cargo-strip
 COPY . .
 
 RUN --mount=type=cache,target=/usr/local/cargo/registry,id=${TARGETPLATFORM} --mount=type=cache,target=/usr/src/edge-runtime/target,id=${TARGETPLATFORM} \
-    GIT_V_TAG=${GIT_V_VERSION} cargo build --profile ${PROFILE} --features "${FEATURES}" && \
-    mv /usr/src/edge-runtime/target/${PROFILE}/edge-runtime /root
-
-RUN objcopy --compress-debug-sections \
-    --only-keep-debug \
-    /root/edge-runtime \
-    /root/edge-runtime.debug
-RUN objcopy --strip-debug \
-    --add-gnu-debuglink=/root/edge-runtime.debug \
-    /root/edge-runtime
-
-RUN ./scripts/install_onnx.sh $ONNXRUNTIME_VERSION $TARGETPLATFORM /root/libonnxruntime.so
-RUN ./scripts/download_models.sh
+  GIT_V_TAG=${GIT_V_VERSION} cargo build --profile ${PROFILE} --features "${FEATURES}" && \
+  cargo strip && \
+  mv /usr/src/edge-runtime/target/${PROFILE}/edge-runtime /root
+
 
```

Comment:
It looks like some contents in the main branch have been reverted.

Reply:
You're right, I did miss theses `objcopy` things 🤦